src/server/scripts/EasternKingdoms/SunwellPlateau/boss_muru.cpp

```diff
@@ -393,54 +376,48 @@ class npc_dark_fiend : public CreatureScript
 
         void Initialize()
         {
-            WaitTimer = 2000;
-            InAction = false;
-        }
+            me->SetDisplayId(me->GetCreatureTemplate()->Modelid2);
```

Comment:
because there is some random between models, sometimes you simply have to force NPC to select exact model, like this one